### ISSUE TITLE
Feedback for Document template - Missing link to the Service variables

### ISSUE THREAD

**Issue by kirstiaa** [SuperOfficeDocs/superoffice-docs#1078]
Hi!
I'm searching for the service template variables, but could not find them anywhere, so it would be good to at least add a link on this page to
https://docs.superoffice.com/en/request/reply-templates/learn/template-variables.html

And also make sure that all variables for Service is added in Docs.
https://help.superoffice.com/Documentation/Help/EN/CRM/UserHelp/index.htm#t=Service%2Ftopics%2FTemplate_variables.html

Would also be good to have some links to this info (I think you have it in Docs somewhere), so it's possible to find them from a logic place.

https://help.superoffice.com/Documentation/Help/EN/CRM/WebHelpAdmin/Quote_templates.pdf?rhhlterm=quote%20template&rhsyns=%20 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.superOffice.com ➟ Docs Team processing.*

* Content Source: [help-en-document-template](https://github.com/SuperOfficeDocs/superoffice-docs/blob/main/docs/en/admin/lists/learn/document-template.md/#L1)

**Comment by digitaldiina** [comment]
I'll add links from this page to the service variables.

We migrated the list of variables for help, and they do have the same number of variables.

We migrated the quote templates PDF to docs more than a year ago. There is a link to the content at the top of the page this issue was reported on.

![image](https://github.com/SuperOfficeDocs/superoffice-docs/assets/23242792/6f6d05d5-e572-41e6-8bce-aefcaeb1ccbe)
